src/main/kotlin/no/nav/syfo/kartleggingssporsmal/domain/KartleggingssporsmalStoppunkt.kt

```diff
@@ -65,10 +66,20 @@ data class KartleggingssporsmalStoppunkt private constructor(
                 !oppfolgingstilfelle.hasTilfelleWithEndMoreThanThirtyDaysAgo()
         }
 
+        /*
+        * Dekker tre caser:
+        * - Det kommer inn informasjon som dytter duration inn i intervallet ila perioden: hasDurationInsideInterval
+        * - Det kommer inn informasjon som dytter duration forbi intervall-slutt, men vi er fortsatt ikke forbi stoppunkttidspunktet i dag (typisk lang sykmelding): willHaveSykedagInIntervalDuringThisPeriod
+        * - Det kommer inn informasjon som sier at duration frem til idag er innenfor intervallet: isInsideIntervalNow
+        */
         private fun oppfolgingstilfelleInsideStoppunktInterval(oppfolgingstilfelle: Oppfolgingstilfelle): Boolean {
-            // TODO: Hva med de som er før start-tidspunkt på intervallet, og så får en veldig lang sykmelding som strekker seg over intervallet?
-            return oppfolgingstilfelle.durationInDays() >= KARTLEGGINGSSPORSMAL_STOPPUNKT_START_DAYS &&
-                oppfolgingstilfelle.durationInDays() <= KARTLEGGINGSSPORSMAL_STOPPUNKT_END_DAYS
+            val durationDays = oppfolgingstilfelle.durationInDays()
+            val durationInDaysUntilNow = ChronoUnit.DAYS.between(oppfolgingstilfelle.tilfelleStart, LocalDate.now()) + 1
+            val hasDurationInsideInterval = durationDays in KARTLEGGINGSSPORSMAL_STOPPUNKT_START_DAYS..KARTLEGGINGSSPORSMAL_STOPPUNKT_END_DAYS
+            val willHaveSykedagInIntervalDuringThisPeriod = durationInDaysUntilNow <= KARTLEGGINGSSPORSMAL_STOPPUNKT_START_DAYS && durationDays >= KARTLEGGINGSSPORSMAL_STOPPUNKT_START_DAYS
+            val isInsideIntervalNow = durationInDaysUntilNow in KARTLEGGINGSSPORSMAL_STOPPUNKT_START_DAYS..KARTLEGGINGSSPORSMAL_STOPPUNKT_END_DAYS
+
+            return hasDurationInsideInterval || willHaveSykedagInIntervalDuringThisPeriod || isInsideIntervalNow
```

Comment:
Kanskje er det tilstrekkelig med den første og den siste (eller har du et eksempel som ikke fanges opp?)?

Reply:
Det er vel den testen `processOppfolgingstilfelle should generate stoppunkt when duration until today is before stoppunkt but new periode sends duration outside interval-end` som feiler hvis denne ikke er på plass.
- Man har vært sykmeldt 6 uker minus 2 dager (40 dager) (er ikke inne i intervallet nå)
- Det kommer inn en sykmelding som er så lang at den dytter `antallSykedager` ut av intervallet.

Så den prøver vel å si at man gjennom den nye informasjonen kommer til å gå over 6 uker, men at man på nåværende tidspunkt ikke gjør det. Hadde man kun sjekket > 6 uker, ville man laget et stoppunkt dersom det kom inn en sykmelding i uke 20 også 🤷🏼‍♂️